src/Avalonia.Controls/ApplicationLifetimes/IActivatableApplicationLifetime.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Avalonia.Controls.ApplicationLifetimes;
+
+public interface IActivatableApplicationLifetime
+{
+    event EventHandler<ActivatedEventArgs> Activated;
+    
+    event EventHandler<ActivatedEventArgs> Deactivated;
```

Comment:
I feel like we should split these args, and add DeactivatedEventArgs instead of reusing. Maybe even with DeactivationKind.

Reply:
If I interpreted correctly what @danwalmsley meant if `ActivationKind.Background` occurs in the Activated event, it indicates that the app has been reactivated from suspension, otherwise in Deactivated it indicates that the app has been suspended. So for each of the elements of the enum.

If so I would suggest renaming:
  ActivatedEventArgs in ApplicationStateEventArgs
  ActivationKind in ApplicationStateKind

and create

  ApplicationStateActivatedEventArgs: ApplicationStateEventArgs
  ApplicationStateDeactivatedEventArgs: ApplicationStateEventArgs

cc @robloo